book/src/proposals/simple-payment-and-state-verification.md

```diff
@@ -89,13 +89,13 @@ LightEntry {
 }
 ```
 
-The light entries are reconstructed from Entries and simply show the entry Merkle Root that was mixed in to the PoH hash, instead of the full transaction set.
+The light entries are reconstructed from Entries and simply show the Entry-Merkle Root that was mixed in to the PoH hash, instead of the full transaction set.
```

Comment:
Consistent capitalization.

Reply:
How about "entry's"?

Reply:
"Entry-Merkle" is used here to discern from "Block-Merkle".  These correspond to diagrams in `art/`

Reply:
Regarding capitalization, the original content went its own way, and that should be corrected first.  There's no precedent for uppercasing and hyphenating new terms.  Merkle is a proper noun, so should be capitalized on its own, but when using it to imply "Merkle hash" in new terms, you should lowercase it. So: "Block-Merkle" should be "block-merkle".